File: Python/sum-of-square-numbers.py
import math

class Solution:
    def judgeSquareSum(self, c):
        """
        :type c: int
        :rtype: bool
        """
        for a in range(int(math.sqrt(c)) + 1):
            b = int(math.sqrt(c - a**2))
            if a**2 + b**2 == c:
                return True

        return False

File: Swift/sum-of-square-numbers.swift
class Solution {
    func judgeSquareSum(_ c: Int) -> Bool {
        for a in 0...Int(sqrt(Double(c))) {
            let b = Int(sqrt(Double(c - a * a)))
            if a * a + b * b == c {
                return true
            }
        }

        return false
    }
}
